webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,17 @@ function BugForm() {
     }, this));
   };
 
+  // See function autoExpand in issues.js
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand",function(){})
+    .on("input.js-autoExpand", "textarea.js-autoExpand", function() {
```

Comment:
I understand the benefits of jQuery event namespaces (`.js-autoExpand` in this case), but we're not using them here. Maybe we can just listen for `input` to make the code easier to understand?